src/js/wizards/Buy/1_Trade/components/OfferTrade.jsx

```diff
@@ -10,19 +10,18 @@ import moment from "moment";
 import escrow from "../../../../features/escrow";
 
 const OfferTrade = ({
-  statusContactCode, name, minToken, maxToken, price: _price, currency, asset, lastActivity,
-  assetQuantity, currencyQuantity, onCurrencyChange, onAssetChange, disabled, t, notEnoughETH, canRelay
-}) => (
-  <Row>
+  statusContactCode, name, minToken, maxToken, price, currency, asset, lastActivity, limitless,
+  assetQuantity, currencyQuantity, onCurrencyChange, onAssetChange, disabled, t, notEnoughETH, canRelay,
+  limitH, limitL, sellerBalance
+}) => {
+  const minFiat = (parseFloat(limitL) / 100).toFixed(2);
+  const maxFiat = (parseFloat(limitH) / 100).toFixed(2);
+  const amountGreaterThanBalance = currencyQuantity > parseFloat(sellerBalance) * price;
```

Comment:
Not sure why you're comparing the fiat amounts, though it probably gives the same result.
Comparing sellerBalance and assetQuantity would have been simpler I think